apps/jubmoji-quest/prisma/schema.prisma

```diff
@@ -39,11 +39,12 @@ model Quest {
 }
 
 model Power {
-  id          Int     @id @default(autoincrement())
-  name        String
-  description String
-  powerType   PowerType
-  powerParams Json   // params for the specific power type
+  id                     Int     @id @default(autoincrement())
+  name                   String
+  description            String
+  sigNullifierRandomness String? // used if we want unique nullifiers for each power
+  powerType              PowerType
+  powerParams            Json    // params for the specific power type
```

Comment:
Can we have a spec for powerParams JSON this in TypeScript in this PR? Or just do it later

Reply:
I think this can go in a separate PR, maybe when we flesh out powers - I image a similar definition to the `ProofClass` setup for powers